apm-sniffer/apm-sdk-plugin/feign-default-http-9.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/feign/http/v9/DefaultHttpClientInterceptor.java

```diff
@@ -81,6 +82,16 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         Tags.HTTP.METHOD.set(span, request.method());
         Tags.URL.set(span, request.url());
         SpanLayer.asHttp(span);
+        if (FeignPluginConfig.Plugin.Feign.TRACE_PARAMS) {
+            if (request.charset() != null && request.body() != null) {
+                String tagValue = new String(request.body(), request.charset());
```

Comment:
This is not just tracking the HTTP parameter, this is tracking the whole request BODY. The config name and document should be adjusted.
Also, you should do this based on HTTP request type, an HTTP request could include binary(such as a file), please use the format in the headers to filter the requests.

Reply:
thank you for your advice . I will try to solve it